update.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 # check for changes in docs and run sphinx
 
+set -e
+set -x
+
```

Comment:
debug?

Reply:
No, I'm leaving it like this deliberately.  One of these is even called out in the commit messages...